src/wow.coffee

```diff
@@ -172,6 +201,11 @@ class @WOW
   resetStyle: ->
     box.style.visibility = 'visible' for box in @boxes
 
+  resetAnimation: (event) ->
+  	if event.type.toLowerCase().indexOf('animationend') >= 0
+  		target = event.target || event.srcElement
+  		target.className = target.className.replace(config.animateClass, '').trim()
```

Comment:
This will leave double spaces if the `animateClass` is not the first or last class. But then again, who cares. :sweat_smile: